Libs/MRML/Core/vtkMRMLProceduralColorNode.cxx

```diff
@@ -129,6 +134,38 @@ void vtkMRMLProceduralColorNode::ProcessMRMLEvents ( vtkObject *caller,
 }
 
 //-----------------------------------------------------------
+vtkLookupTable* vtkMRMLProceduralColorNode::GetLookupTable()
+{
+  this->ConvertedCTFtoLUT->SetNumberOfTableValues(0);
+
+  // since setting the range is a no-op on color transfer functions,
+  // copy into a color look up table with, for now 256 entries
+  const int numEntries = 256;
```

Comment:
Make it number of color table entries a member variable, initialized to 256 by default. No need to add GUI to change it.